cmd/olm/main.go

```diff
@@ -69,6 +69,9 @@ var (
 
 	namespace = pflag.String(
 		"namespace", "", "namespace where cleanup runs")
+
+	disableCopiedCSVs = pflag.Bool(
+		"disableCopiedCSVs", false, "disables copied CSVs in AllNamespace mode when enabled")
```

Comment:
It looks like there's already precedence for camel-cased flag names poking around the flag definitions listed above - my preference is certainty towards [unix-style](http://www.catb.org/~esr/writings/taoup/html/ch10s05.html), e.g. --disable-coped-csvs. Any thoughts on that style vs. continuing to use this existing style?

Reply:
A couple of more questions I had:

- How would this CLI flag interact with the OLMConfig API?
- Any thoughts on a feature gating implementation vs. a CLI flag?

Reply:
> How would this CLI flag interact with the OLMConfig API?
This behavior is defined in the enhancement and warrented [some discussion](https://github.com/operator-framework/enhancements/pull/92/files#r750593846). Personally, I would like to remove the flag. Thoughts @joelanford @njhale @joelanford @timflannagan?

Reply:
>Any thoughts on a feature gating implementation vs. a CLI flag?

As of the latests version of this PR, this can be configured via the [olmConfig CRD](https://github.com/operator-framework/api/pull/198).

Reply:
>It looks like there's already precedence for camel-cased flag names poking around the flag definitions listed above - my preference is certainty towards unix-style, e.g. --disable-coped-csvs. Any thoughts on that style vs. continuing to use this existing style?

I prefer the unix style as well, but wanted to remain consistent within the project. If we drop support for the flag this is no longer an issue.

Reply:
I'm largely a fan of feature flagging but if we have another API that can drive those decisions, it's just extra complexity at the end of the day.

Reply:
I'd be okay with either of these:
1. change the name to `defaultDisableCopiedCSVs`, which implies it can/will be overridden by the OLMConfig object if set there.
2. just remove the flag

The only argument I have against (2) is that if you're a user deploying OLM with the intention of enabling the flag, it's less work to do it as a CLI flag. This probably really only applies to OLM developers, so maybe the value proposition isn't there.

Also, fwiw, I prefer unix-style, but I prefer consistency more. If we want to change to unix-style, we should deprecate all of the camelCase flags and add the unix-style variants, IMO.

Reply:
> The only argument I have against (2) is that if you're a user deploying OLM with the intention of enabling the flag, it's less work to do it as a CLI flag. This probably really only applies to OLM developers, so maybe the value proposition isn't there.

Although removing the flag adds a bit of a burden on the developer, there are benefits to having a single point of entry for toggling this feature:
1. It ensures that developers and users rely on the same workflow.
2. The flag can always be added later if there is demand for it. Removing the flag once released is much more difficult.
3. It isn't that hard to enable the feature via the API:
```
$ kubectl create -f -<<EOF
apiVersion: operators.coreos.com/v1
kind: OLMConfig
metadata:
  name: cluster
spec:
  features:
    disableCopiedCSVs: true
EOF
```

Reply:
@awgreene If we're going to run down this direction of having the API drive all runtime configurations, do we need to also update the EP? How would users enable this configuration in downstream environments?

Reply:
>@awgreene If we're going to run down this direction of having the API drive all runtime configurations, do we need to also update the EP? 

Yes @timflannagan , I have a [WIP PR](https://github.com/operator-framework/enhancements/pull/103) for the enhancement and will capture these changes there

>How would users enable this configuration in downstream environments?

We will add the cluster olmConfig to the `/manifests` dir and add the `release.openshift.io/create-only: "true"` annotation, allowing users to enable/disable the feature.

Reply:
> We will add the cluster olmConfig to the /manifests dir with the release.openshift.io/create-only: "true" annotation, allowing users to enable/disable the feature.

+1 that's exactly what I had in mind as well.